### DISCUSSION TITLE
Can't deploy after `ALTER TABLE mytable AUTO_INCREMENT=1;`

### DISCUSSION THREAD

**Discussion by danaugrs** [planetscale/discussion#78] [Bug Reports] [answered]
I created a branch, executed `ALTER TABLE mytable AUTO_INCREMENT=1;` and created a deploy request.
When I tried to deploy I got an error:

```
> pscale deploy-request deploy prod 2
Error: This deploy request cannot be deployed.
```

The UI shows me:

![Screenshot from 2021-08-19 14-47-20](https://user-images.githubusercontent.com/5261448/130118699-0baaca93-3685-434d-8a48-f2780476a95b.png)

TLDR: I can't execute `ALTER TABLE mytable AUTO_INCREMENT=1;` in the main branch since it's DDL and I can't merge a branch with that change.

**Comment by jgreet** [comment, marked as answer]
Hi @danaugrs. You're right, that `ALTER` is DDL and can't be applied in a production branch via a deploy request. The auto-increment counter value isn't included when we diff the branch schemas to create a deploy request. 

**One solution:** you can disable safe migrations for your production branch. This will allow you to run the DDL directly in production without using a deploy request.

~At the moment we don't have a workaround that can apply this change to a production branch.~

**Reply by danaugrs** [reply]
Hi @jgreet, I was deleting all rows and resetting `AUTO_INCREMENT` on one of my tables prior to doing a data seed.
I ended up dropping the table in one deploy request then recreating it in another.
Maybe this is not a very common use case, but it's a bit worrying that there are some DDL statements that can't be applied to a production branch.

**Comment by danaugrs** [comment]
Any updates to this? Or plans to allow such DDL statements? The only way to reset this currently is to delete the database and recreate.

**Reply by Brian-1150** [reply]
You can go to your production branch and disable "safe migrations".  Then you can execute any DDL statement you want
